tv.kodi.Kodi.json

```diff
@@ -0,0 +1,123 @@
+{
+    "app-id": "tv.kodi.Kodi",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk10"
+    ],
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=xdg-music",
+        "--filesystem=xdg-pictures",
+        "--filesystem=xdg-videos",
+        "--share=ipc",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--socket=x11",
+        "--system-talk-name=org.bluez",
+        "--system-talk-name=org.freedesktop.Avahi"
+    ],
+    "command": "kodi",
+    "rename-desktop-file": "kodi.desktop",
+    "rename-icon": "kodi",
+    "modules": [
+        "shared-modules/udev/udev-175.json",
+        "shared-modules/python2.7/python-2.7.15.json",
+        "deps/avahi.json",
+        "deps/libical.json",
+        "deps/bluez.json",
+        "deps/fmt.json",
+        "deps/fribidi.json",
+        "deps/fstrcmp.json",
+        "deps/ghostscript.json",
+        "shared-modules/glu/glu-9.0.0.json",
+        "deps/libass.json",
+        "deps/libcdio.json",
+        "deps/lzo2.json",
+        "deps/samba.json",
+        "deps/swig.json",
+        "deps/taglib.json",
+        "deps/tinyxml.json",
+        {
+            "name": "kodi",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DVERBOSE=1",
+                "-DENABLE_INTERNAL_FFMPEG=ON",
+                "-DENABLE_INTERNAL_FLATBUFFERS=ON",
+                "-DENABLE_INTERNAL_RapidJSON=ON",
+                "-DFLATBUFFERS_URL=build/download/flatbuffers-1.9.0.tar.gz",
+                "-DCROSSGUID_URL=build/download/crossguid-8f399e8bd4.tar.gz",
+                "-DRapidJSON_URL=build/download/rapidjson-1.1.0.tar.gz",
+                "-DFFMPEG_URL=build/download/4.0.2-Leia-Alpha3.tar.gz",
+                "-DLIBDVDCSS_URL=build/download/1.4.1-Leia-Beta-3.tar.gz",
+                "-DLIBDVDREAD_URL=build/download/6.0.0-Leia-Alpha-3.tar.gz",
+                "-DLIBDVDNAV_URL=build/download/libdvdnav-6.0.0-Leia-Alpha-3.tar.gz",
+                "-DJava_JAVA_EXECUTABLE=/usr/lib/sdk/openjdk10/bin/java"
+            ],
+            "post-install": [
+                "sed -i 's|^KODI_DATA.*|KODI_DATA=${HOME}/.var/app/tv.kodi.Kodi/data|' /app/bin/kodi"
+            ],
```

Comment:
Add binary addons. I have send you a pull request (no. 4)

Reply:
You are sharing network during build, which I don't like. It makes the build system download files and that should be avoided. Is there a way we can download them before instead?

Reply:
The main problem is that it uses a lot of reverences which will be downloaded automatacly. I don't know if Kodi has an archive version with all references, but the format looks like that a script parse it and could generate the source list with makefile, etc. .